contracts/modules/StakeModule/libs/StakeModuleLib.sol

```diff
@@ -885,9 +885,10 @@ library StakeModuleLib {
     uint256 poolId
   ) public view returns (bool isValid) {
     uint256 lastupdate = self.gETH.priceUpdateTimestamp(poolId);
+    // @dev IMPORTANT: || => &&
     unchecked {
       isValid =
-        lastupdate + PRICE_EXPIRY >= block.timestamp &&
+        lastupdate + PRICE_EXPIRY >= block.timestamp ||
```

Comment:
This change should be reverted before merging. This is just for the Holesky network.